rmgpy/cantherm/thermo.py

```diff
@@ -188,6 +188,30 @@ def save(self, outputFile):
                 f.write('\n')
         f.close()
 
+        return chem_string
+
+    def save_species_to_database_file(self, path):
```

Comment:
I think this may be more appropriate as a method of `CanthermSpecies`. Let me know what you think.

Reply:
I agree